tests/nunit_tools/nunit_meta_runner/Program.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.Loader;
+using System.Collections.Generic;
+using System.Diagnostics;
+using NUnitLite;
+using NUnit.Framework.Api;
+using NUnit.Framework.Interfaces;
+
+// preloading dependencies
+using Trinity;
+using NUnit.Framework;
+
+namespace NUnitMetaRunner
+{
+    class Program
+    {
+        public static int Main(string[] args)
+        {
+            if (args.Length < 2)
+            {
+                Console.Error.WriteLine("Usage: runner <runner-path> <test-assembly> <result-dir> [nunitlite-arg ..]");
```

Comment:
The testing process for one test assembly is:

1. Meta-runner scans the test assembly and get the list of tests.
2. Meta-runner starts a test-runner for each test in the test assembly.
3. (TODO) Merge the result XML files into one.

... where, the path of the test-runner is specified in runner-path.